testing-common/src/testFixtures/java/org/hypertrace/agent/testing/TestAgentStarter.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright The Hypertrace Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.hypertrace.agent.testing;
+
+import io.opentelemetry.javaagent.bootstrap.AgentStarter;
+
+final class TestAgentStarter implements AgentStarter {
+
+  @Override
+  public boolean delayStart() {
+    return false;
+  }
+
+  @Override
+  public void start() {}
+
+  @Override
+  public ClassLoader getExtensionClassLoader() {
+    return ClassLoader.getSystemClassLoader();
```

Comment:
we do this to instruct the agent to load extensions from the system class loader - we won't need to do this once we migrate to their build plugins